spec/swagger.json

```diff
@@ -0,0 +1,110 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Sic-Code-API",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "http://api.chs.local/siccode-api",
+      "description": "Development server"
+    }
+  ],
+  "paths": {
+    "/search": {
+      "post": {
+        "summary": "Search for sic code via economic activity",
+        "operationId": "searchSicCodes",
+        "description": "Return all matching sic code records that correspond with the search terms. This will be ordered by most number of hits (for example, results with 2 keyword matches will be retuned before matches with 1 keyword match)",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "keywords",
+            "description": "Keywords for the economic activities to be used in the search",
```

Comment:
One variable containing many values?
This seems to be one parameter whose key could point to many values in one string. If so, how are they separated?
Perhaps consider this instead:
https://stackoverflow.com/questions/62527254/how-does-open-api-3-0-support-a-single-query-param-key-with-multiple-values

Reply:
Also, this is perhaps the one parameter that could do with an example.

Reply:
Updated this variable to better illustrate it as a search string. See commit